kuzzleio/kuzzle
# Remove import mappings lock on `admin:resetDatabase` action

#2297

# How do you test it

1. Create new app
2. In `app.ts`:
  ```js
  import { Backend } from './kuzzle';

  const app = new Backend('issue-2341');

  app.import.mappings({
    foo: {
      bar: {
        mappings: {
          properties: {
            first_mapping: { type: 'keyword'}
          }
        }
      }
    }
  });
  ```
3. Execute: `kourou admin:resetDatabase`
4. Replace in `app.ts`: `first_mapping` by `second_mapping`
5. Check mappings, `second_mapping` will be there and not `first_mapping`
